src/components/inputs/async-creatable-select-input/async-creatable-select-input.js

```diff
@@ -59,7 +61,7 @@ const AsyncCreatableSelectInput = props => {
           inputId={props.id}
           inputValue={props.inputValue}
           isClearable={props.isClearable}
-          isDisabled={props.isDisabled}
+          isDisabled={props.isDisabled || props.isReadOnly}
```

Comment:
What's the benefit of the prop if it maps to this?

Reply:
`react-select` doesn't support `isReadOnly` because it doesn't have any functional difference from `isDisabled`, it's only a matter of styling it differently. So basically I'm disabling it internally and giving it the same `read-only` visuals as we have for the other inputs.

But the main point of adding the prop is because almost all of our inputs support `readOnly` and users of the library can erroneously use it for this component as well, as it happened [here](https://github.com/commercetools/merchant-center-frontend/pull/7977). So it's a matter of consistency.